src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -0,0 +1,720 @@
+
+        if os.path.exists(os.path.join(getWMBASE(),
+                                       'src/python/WMCore/WMRuntime/Unpacker.py')):
+            self.unpacker = os.path.join(getWMBASE(),
+                                         'src/python/WMCore/WMRuntime/Unpacker.py')
+        else:
+            self.unpacker = os.path.join(getWMBASE(),
+                                         'WMCore/WMRuntime/Unpacker.py')
+
+        self.agent = getattr(config.Agent, 'agentName', 'WMAgent')
+        self.sandbox = None
+        self.scriptFile = None
+        self.submitDir = None
+        self.submitWMSMode = getattr(config.BossAir, 'submitWMSMode', False)
+        self.defaultTaskPriority = getattr(config.BossAir, 'defaultTaskPriority', 0)
+        self.maxTaskPriority = getattr(config.BossAir, 'maxTaskPriority', 1e7)
+        self.jobsPerSubmit = getattr(config.JobSubmitter, 'jobsPerSubmit', 200)
+        self.deleteJDLFiles = getattr(config.JobSubmitter, 'deleteJDLFiles', True)
+
+        # Required for global pool accounting
+        self.acctGroup = getattr(config.BossAir, 'acctGroup', "production")
+        self.acctGroupUser = getattr(config.BossAir, 'acctGroupUser', "cmsdataops")
+
+        # Build a requirement string
+        self.reqStr = "stringListMember(GLIDEIN_CMSSite, DESIRED_Sites) && ((REQUIRED_OS=?=\"any\") || (GLIDEIN_REQUIRED_OS=?=REQUIRED_OS))"
+        if hasattr(config.BossAir, 'condorRequirementsString'):
+            self.reqStr = config.BossAir.condorRequirementsString
+
+        return
+
+    def submit(self, jobs, info=None):
+        """
+        _submit_
+
+
+        Submit jobs for one subscription
+        """
+
+        # If we're here, then we have submitter components
+        self.scriptFile = self.config.JobSubmitter.submitScript
+        self.submitDir = self.config.JobSubmitter.submitDir
+        timeout = getattr(self.config.JobSubmitter, 'getTimeout', 400)
+
+        successfulJobs = []
+        failedJobs = []
+        jdlFiles = []
+
+        if len(jobs) == 0:
+            # Then was have nothing to do
+            return successfulJobs, failedJobs
+
+        if not os.path.exists(self.submitDir):
+            os.makedirs(self.submitDir)
+
+        # Submit the jobs
+        for jobsReady in grouper(jobs, self.jobsPerSubmit):
+
+            idList = [x['id'] for x in jobsReady]
+            jdlList = self.makeSubmit(jobList=jobsReady)
+            if not jdlList:
+                # Then we got nothing
+                logging.error("No JDL file made!")
+                return {'NoResult': [0]}
+            jdlFile = "%s/submit_%i_%i.jdl" % (self.submitDir, os.getpid(), idList[0])
+
+            with open(jdlFile, 'w') as handle:
+                handle.writelines(jdlList)
+            jdlFiles.append(jdlFile)
+
+            # Now submit them
+            logging.info("About to submit %i jobs", len(jobsReady))
+
+            command = "condor_submit --terse -disable %s" % jdlFile
```

Comment:
Looked into it. There isn't enough documentation to do this. I took an example jdl file and dumped it into classads as described in the documentation. Where I am stuck is this:

"""Then, load the resulting contents of <filename> into Python. """

How ? I guess I need to convert this into a classad instance somehow, but

a) I don't want to go via a file if I can avoid it
b) with bulk submission there are lots of duplicate parameters since they differ for different jobs

Any example code how to do this ?